### PR TITLE
docs(fx114): inverted-colors reference page and relnote updates

### PR DESCRIPTION
**Pull request by dipikabh**
### Description

Fx114 adds support for the [`inverted-colors`](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/inverted-colors) media feature.
The support is behind the `layout.css.inverted-colors.enabled` flag, which is `false` by default.

This PR updates the existing content page and adds an entry to the the experimental features page.

On the content page:
- The "Experimental banner" (`{{SeeCompatTable}}`) is added because the feature is not supported in any browser and supported in Firefox only now.
- A few edits have been done for clarity and for aligning with the template.

### Additional details

Spec: https://drafts.csswg.org/mediaqueries-5/#inverted

### Related issues and pull requests

Bug: https://bugzilla.mozilla.org/show_bug.cgi?id=1794628
Doc issue: https://github.com/mdn/content/issues/26691
BCD PR: https://github.com/mdn/browser-compat-data/pull/19881

### CODE REVIEW

File: files/en-us/web/css/@media/inverted-colors/index.md [hunk 1]
```diff
@@ -5,35 +5,47 @@ page-type: css-media-feature
...
-  If you're using inverted colors, this text should be blue on white (the
-  inverse of yellow on black). If you're not, it should be red on light gray.
+  If color inversion is detected, this text will appear blue on white (the
+  inverse of yellow on black). If no color inversion is happening, the text
+  will appear red on light gray.
 </p>
 <p>
-  If the text is gray, your browser doesn't support the `inverted-colors` media
-  feature.
+  If the text is gray, then it means that your browser doesn't support the
```
Comment (estelle) [inline files/en-us/web/css/@media/inverted-colors/index.md:43]: I prefer the original 

```suggestion
  If the text is gray, it means your browser doesn't support the
```

I recommend adding some other effect, like an overline, for people with color blindness or cognitive issues; while generally this media query will be used to change colors, our explanations can add a feature for teaching reasons.

File: files/en-us/mozilla/firefox/experimental_features/index.md [hunk 2]
```diff
@@ -800,6 +800,47 @@ See ([Firefox bug 1736914](https://bugzil.la/1736914)) for more details.
   </tbody>
 </table>
 
+### inverted-colors media feature
+
+The CSS [`inverted-colors`](/en-US/docs/Web/CSS/@media/inverted-colors) media feature lets you detect if a user agent or teh underlying operating system is inverting colors.
```
Comment (estelle) [inline files/en-us/mozilla/firefox/experimental_features/index.md:805]:
```suggestion
The CSS [`inverted-colors`](/en-US/docs/Web/CSS/@media/inverted-colors) media feature lets you detect if a user agent or the underlying operating system is inverting colors.
```

File: files/en-us/web/css/@media/inverted-colors/index.md [hunk 3]
```diff
@@ -5,35 +5,47 @@ page-type: css-media-feature
 browser-compat: css.at-rules.media.inverted-colors
 ---
 
-{{CSSRef}}
+{{CSSRef}}{{SeeCompatTable}}
 
-The **`inverted-colors`** [CSS](/en-US/docs/Web/CSS) [media feature](/en-US/docs/Web/CSS/@media#media_features) can be used to test whether the {{glossary("user agent")}} or underlying OS is inverting colors.
+The **`inverted-colors`** [CSS](/en-US/docs/Web/CSS) [media feature](/en-US/docs/Web/CSS/@media#media_features) is used to test if the {{glossary("user agent")}} or the underlying operating system has inverted all colors. Inversion of colors can unpleasant side effects, such as shadows turning into highlights, which can reduce the readability of the content. You can detect if inversion is happening by using this media feature and style the content.
```
Comment (estelle) [inline files/en-us/web/css/@media/inverted-colors/index.md:10]:
```suggestion
The **`inverted-colors`** [CSS](/en-US/docs/Web/CSS) [media feature](/en-US/docs/Web/CSS/@media#media_features) is used to test if the {{glossary("user agent")}} or the underlying operating system has inverted all colors. Inversion of colors can have unpleasant side effects, such as shadows turning into highlights, which can reduce the readability of the content. Using this media feature, you can detect if inversion is happening and style the content accordingly.
```

File: files/en-us/web/css/@media/inverted-colors/index.md [hunk 4]
```diff
@@ -5,35 +5,47 @@ page-type: css-media-feature
...
 - `inverted`
-  - : All pixels within the displayed area have been inverted.
+  - : Indicates that all pixels within the displayed area have been inverted. This keyword value evaluates as true.
 
 ## Examples
 
-### HTML
+### Applying styles if color inversion is detected
+
+The example below shows the effects with both the keyword values of the `inverted-colors` media feature and also when the `inverted-colors` media feature is not supported.
```
Comment (estelle) [inline files/en-us/web/css/@media/inverted-colors/index.md:32]:
```suggestion
This example demonstrates the effects of both `inverted-colors` media feature keyword values and when the `inverted-colors` media feature is not supported.
```

**Review by estelle — COMMENTED** [review]
suggest adding more than just color to demonstrate the effects for readers with color blindness and / or cognition differences

**Comment by dipikabh** [comment]
Thanks for the review @estelle. I've incorporated your feedback.

File: files/en-us/web/css/@media/inverted-colors/index.md [hunk 5]
```diff
@@ -5,35 +5,48 @@ page-type: css-media-feature
 browser-compat: css.at-rules.media.inverted-colors
 ---
 
-{{CSSRef}}
+{{CSSRef}}{{SeeCompatTable}}
 
-The **`inverted-colors`** [CSS](/en-US/docs/Web/CSS) [media feature](/en-US/docs/Web/CSS/@media#media_features) can be used to test whether the {{glossary("user agent")}} or underlying OS is inverting colors.
+The **`inverted-colors`** [CSS](/en-US/docs/Web/CSS) [media feature](/en-US/docs/Web/CSS/@media#media_features) is used to test if the {{glossary("user agent")}} or the underlying operating system has inverted all colors. Inversion of colors can have unpleasant side effects, such as shadows turning into highlights, which can reduce the readability of the content. Using this media feature, you can detect if inversion is happening and style the content accordingly.
```
Comment (estelle) [inline files/en-us/web/css/@media/inverted-colors/index.md:10]:
```suggestion
The **`inverted-colors`** [CSS](/en-US/docs/Web/CSS) [media feature](/en-US/docs/Web/CSS/@media#media_features) is used to test if the {{glossary("user agent")}} or the underlying operating system has inverted all colors. 

Inversion of colors can have unpleasant side effects, such as shadows turning into highlights, which can reduce the readability of the content. Using this media feature, you can detect if inversion is happening and style the content accordingly while respecting user preference.
```

File: files/en-us/web/css/@media/inverted-colors/index.md [hunk 6]
```diff
@@ -5,35 +5,48 @@ page-type: css-media-feature
...
-  inverse of yellow on black). If you're not, it should be red on light gray.
+  If color inversion is detected, this text will appear blue on white (the
+  inverse of yellow on black) along with a line over the text. If no color
+  inversion is happening, the text will appear red on light gray without
+  the line over the text.
 </p>
 <p>
-  If the text is gray, your browser doesn't support the `inverted-colors` media
-  feature.
+  If the text is gray, it means your browser doesn't support the
```
Comment (estelle) [inline files/en-us/web/css/@media/inverted-colors/index.md:44]:
```suggestion
  If the text is gray and no overline is present, it means your browser doesn't support the
```

File: files/en-us/web/css/@media/inverted-colors/index.md [hunk 7]
```diff
@@ -69,5 +83,5 @@ p {
 
 ## See also
 
-- [Using Media Queries](/en-US/docs/Web/CSS/Media_Queries/Using_media_queries)
 - [@media](/en-US/docs/Web/CSS/@media)
+- [Using media queries](/en-US/docs/Web/CSS/Media_Queries/Using_media_queries)
```
Comment (estelle) [inline files/en-us/web/css/@media/inverted-colors/index.md:87]:
```suggestion
- [CSS media queries](/en-US/docs/Web/CSS/Media_Queries) module
- [Using media queries](/en-US/docs/Web/CSS/Media_Queries/Using_media_queries)
```

File: files/en-us/web/css/@media/inverted-colors/index.md (hunk 5 above)
Comment (estelle) [reply files/en-us/web/css/@media/inverted-colors/index.md:10]: suggesting adding the paragraph break as the first paragraph is the summary used in multiple applications like devtools.

**Review by estelle — COMMENTED** [review]
3 more suggestions

File: files/en-us/web/css/@media/inverted-colors/index.md (hunk 5 above)
Comment (dipikabh) [reply files/en-us/web/css/@media/inverted-colors/index.md:10]: 👍 Thanks

File: files/en-us/web/css/@media/inverted-colors/index.md [hunk 8]
```diff
@@ -5,35 +5,50 @@ page-type: css-media-feature
 browser-compat: css.at-rules.media.inverted-colors
 ---
 
-{{CSSRef}}
+{{CSSRef}}{{SeeCompatTable}}
 
-The **`inverted-colors`** [CSS](/en-US/docs/Web/CSS) [media feature](/en-US/docs/Web/CSS/@media#media_features) can be used to test whether the {{glossary("user agent")}} or underlying OS is inverting colors.
+The **`inverted-colors`** [CSS](/en-US/docs/Web/CSS) [media feature](/en-US/docs/Web/CSS/@media#media_features) is used to test if the {{glossary("user agent")}} or the underlying operating system has inverted all colors.
```
Comment (dipikabh) [inline files/en-us/web/css/@media/inverted-colors/index.md:10]:
```suggestion
The **`inverted-colors`** [CSS](/en-US/docs/Web/CSS) [media feature](/en-US/docs/Web/CSS/@media#media_features) is used to test if the {{glossary("user agent")}} or the underlying operating system has inverted all colors.
```

**Comment by dipikabh** [comment]
> 3 more suggestions

Thanks @estelle, these are all valid comments and I've fixed them. This is ready for a final look and approval.

**Review by estelle — APPROVED** [review]
Yay! 🎉

**Comment by dipikabh** [comment]
Thanks, @estelle!

**Pull request merged**